app/views.py

```diff
@@ -177,3 +186,26 @@ def edit_post_view(request, primary_key):
             })
     except IntegrityError:
         return HttpResponseBadRequest()
+
+
+def comment(request):
+    if request.method == 'POST':
+        post_id = request.POST.get('post_id')
+        content = request.POST.get('comment_content')
+
+        if content is None or content == '':
+            messages.error(request, _('Nội dung bình luận không được để trống!'))
+            return redirect('post_detail', primary_key=post_id)
+            
+        parent_id = request.POST.get('parent_id')
+
+        post = Post.objects.get(pk=post_id)
```

Comment:
dùng thằng function `get_object_or_404` để xử lý nếu ko tìm thấy bản ghi nhé
```suggestion
        post = get_object_or_404(Post, pk=post_id)
```